tutorials/30_File_Type_Preprocessing_Index_Pipeline.ipynb

```diff
@@ -0,0 +1,1328 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    pip install transformers[torch,sentencepiece]==4.32.1 sentence-transformers&gt;=2.2.0
</pre><p>Do we need to install <code>transformers[torch,sentencepiece]</code> for this notebook? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/274/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>nope! Removed. Good catch.</p>